### DISCUSSION TITLE
Some vocoder questions

### DISCUSSION THREAD

**Discussion by jhlfrfufyfn** [coqui-ai/TTS#1006] [General Q&A] [answered]
Hello everyone!

I want to train a vocoder on a small 4 hour dataset in Belarusian to try to improve sound generation quality of my model, but I'm kind of on a deadline here. I tried to find some information about the time needed to train, but I didn't really find anything. 

So I want to know a couple of things:
1. What vocoder takes the least time to train? 
2. How much time approximately would it take to train it?
3. Would it be better than Griffin-Lim Vocoder?

Thanks in advance!

**Comment by erogol** [comment, marked as answer]
What vocoder takes the least time to train? - HifiGAN, Univnet
How much time approximately would it take to train it?  -  2 days to 1 week based on data quality
Would it be better than Griffin-Lim Vocoder? - Yes
